BaristaSwift/Common/Common/Base/Store.swift

```diff
@@ -0,0 +1,223 @@
+//
+//  Store.swift
+//  Test
+//
+//  Created by David Vieser on 9/27/17.
+//  Copyright © 2017 Salesforce. All rights reserved.
+//
+
+import Foundation
+import SalesforceSDKCore
+import SalesforceSwiftSDK
+import SmartStore
+import SmartSync
+
+public typealias SyncCompletion = ((SFSyncState?) -> Void)?
+
+public class Store<objectType: StoreProtocol> {
+
+    private final let pageSize: UInt = 100
+    
+    public init() {
+//        self.store.removeAllSoups()
```

Comment:
Remove commented out code.

Reply:
I'll keep the commented code around - it might be useful when debugging the app.

Reply:
Sounds good.